### PR TITLE
Add Winfo.app v1.0

### PR DESCRIPTION
**Pull request by artdevjs**
After making all changes to the cask:
- [x] `brew cask audit --download winfo` is error-free.
- [x] `brew cask style --fix winfo` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:
- [x] Named the cask according to the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask install winfo` worked successfully.
- [x] `brew cask uninstall winfo` worked successfully.
- [x] Checked there are no [open pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked that the cask was not already refused in [closed issues](https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed).

### CODE REVIEW

File: Casks/winfo.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask 'winfo' do
+  version '1.0'
+  sha256 'a80dd06aa0824b93040d8098591d26f8bd3e338a365da62a114ae49204c011a1'
+
+  url 'http://www.irradiatedsoftware.com/download/Winfo.zip'
```
Comment (vitorgalvao) [inline Casks/winfo.rb]: Appcast is `http://www.irradiatedsoftware.com/updates/profiles/profile.php`.

File: Casks/browser-chooserx.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+cask 'browser-chooserx' do
```
Comment (vitorgalvao) [inline Casks/browser-chooserx.rb]: This file must be remove (is on the other PR).

File: Casks/winfo.rb (hunk 1 above)
Comment (vitorgalvao) [inline Casks/winfo.rb]: Also add `auto_updates true`.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by ArtemMedvedev** [comment]
@vitorgalvao done

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**